app/controllers/todo_lists_controller.rb

```diff
@@ -0,0 +1,51 @@
+class TodoListsController < ApplicationController
+    skip_forgery_protection
```

Comment:
Was it necessary to add this?

Reply:
Actually i was getting the below error, so to handle this case i have added that during post and update  :
```
ActionController::InvalidAuthenticityToken in UsersController#create
Can't verify CSRF token authenticity.
```

Reply:
Check if we can specify this in some config file.